gax-grpc/src/main/java/com/google/api/gax/grpc/OperationFuture.java

```diff
@@ -121,58 +127,19 @@ public ResponseT get(long timeout, TimeUnit unit)
     return resultFuture.get();
   }
 
-  /**
```

Comment:
I see your point, but we haven't been doing that as a practice. I might actually prefer doing a separate PR to add `{@link}`s to all of our `@Override` methods.